app/services/shared_markers.rb

```diff
@@ -19,7 +19,8 @@ def process_link(link, text = nil)
                               })
     end
 
-    content_tag(:a, text.nil? ? link : text, options)
+    # Marking the text content as HTML safe as <tt>content_tag</tt> already escapes it for us
+    content_tag(:a, text.nil? ? link : text.html_safe, options)
```

Comment:
Irrelevant here because it's already escaped. 

![](https://user-images.githubusercontent.com/1809170/178102715-b3b4c053-2a38-4001-908d-b9e747769fb1.jpg)